winbuild/config.py

```diff
@@ -3,12 +3,13 @@
 SF_MIRROR = 'http://iweb.dl.sourceforge.net'
 PILLOW_DEPENDS_DIR = 'C:\\pillow-depends\\'
 
-pythons = {  # '26': 7,
-           '27': 7,
-           'pypy2': 7,
-           # '32': 7,
-           '33': 7.1,
-           '34': 7.1}
+pythons = {'27': {'compiler':7, 'vc':2008},
+           'pypy2': {'compiler':7, 'vc':2008},
+           '33': {'compiler':7.1, 'vc':2010},
```

Comment:
Shall we remove this line? It's not used and EOL Python 3.3 was removed in 5.0.0 (2018-01-01).

Reply:
Okay, done